controllers/helpdesk.js

```diff
@@ -6,62 +6,29 @@ const api = require('../api');
-        let content = {
-            "text": "User: " + user.displayName + "\n"
-            + "E-Mail: " + email + "\n"
-            + "Schule: " + res.locals.currentSchoolData.name + "\n"
-            + innerText
-            + "User schrieb folgendes: \n" + req.body.content.text
-        };
-        req.body.content = content;
-        req.body.headers = {
-            "X-Zammad-Customer-Email": email
-        };
-        api(req).post('/mails', {json: req.body}).then(_ => {
-            res.sendStatus(200);
-        }).catch(err => {
-            res.status((err.statusCode || 500)).send(err);
-        });
-    } else {
-        api(req).post('/helpdesk', {
-            json: {
-                subject: req.body.subject,
-                category: req.body.category,
-                currentState : req.body.currentState,
-                targetState: req.body.targetState,
-                userId: res.locals.currentUser._id,
-                schoolId: res.locals.currentSchoolData._id
-            }
-        }).then(_ => {
-            api(req).get('/users', {qs: { roles: ['helpdesk']}})
-            .then(data => {
-                data.data.map(user => {
-                    if (process.env.NOTIFICATION_SERVICE_ENABLED) {
-                        api(req).post('/notification/messages', {
-                            json: {
-                                "title": "Ein neues Problem wurde gemeldet.",
-                                "body": "",
-                                "token": user._id,
-                                "priority": "high",
-                                "action": `${(req.headers.origin || process.env.HOST)}/administration/helpdesk`,
-                                "scopeIds": [
-                                    user._id
-                                ]
-                            }
-                        });
-                    }
-                });
-            });
-            res.sendStatus(200);
-        }).catch(err => {
-            res.status((err.statusCode || 500)).send(err);
-        });
-    }
+    api(req).post('/helpdesk', {
+        json: {
+            type: req.body.type,
+            subject: req.body.subject,
+            category: req.body.category,
+            role: req.body.role,
+            desire: req.body.desire,
+            benefit: req.body.benefit,
+            acceptanceCriteria: req.body.acceptanceCriteria,
+            currentState : req.body.currentState,
+            targetState: req.body.targetState,
+            schoolName: res.locals.currentSchoolData.name,
+            userId: res.locals.currentUser._id,
+            email: res.locals.currentUser.email ? res.locals.currentUser.email : "",
+            schoolId: res.locals.currentSchoolData._id,
+            cloud: res.locals.theme.title
+        }
+    })
+    .then(_ => {
+        res.sendStatus(200);
+    }).catch(err => {
+        res.status((err.statusCode || 500)).send(err);
```

Comment:
New: Please add Winston Logging to all not-positive (not only catch if there are other error handlings) handlings. 
Example:
```
const logger = require('winston');
 }).catch(err => {
        logger.warn(err);   
```
You can also pass just a string like warn("Error on user select").